frontend/src/app/testResults/uploads/CsvSchemaDocumentation.tsx

```diff
@@ -128,68 +175,161 @@ const CsvSchemaDocumentation = () => {
             title="SimpleReport bulk results uploader tutorial video"
             src="https://www.youtube.com/embed/qqhC7PFBdek"
           />
-          <p>
-            <strong>In this guide</strong>
-          </p>
+          <h2>In this guide</h2>
           <ul>
             <li>
-              <a href="#preparing-upload" className="usa-link">
-                Preparing and uploading a spreadsheet
+              <a href="#formatting-guidelines" className="usa-link">
+                Data formatting guide
               </a>
             </li>
             <li>
-              <a href="#formatting-guidelines" className="usa-link">
-                General formatting guidelines
+              <a href="#preparing-upload" className="usa-link">
+                Preparing and uploading a spreadsheet
               </a>
-              {schema.fields.map((field, fieldIndex) => {
-                return (
-                  <ol key={`toc-${fieldIndex}`} className="">
-                    {field.sections?.map((section, sectionIndex) => {
-                      return (
-                        <li key={`toc-${fieldIndex}-${sectionIndex}`}>
-                          <a href={`#${section.slug}`} className="usa-link">
-                            {section.title}
-                          </a>
-                        </li>
-                      );
-                    })}
-                  </ol>
-                );
-              })}
             </li>
           </ul>
-          <p>
-            <strong>Resources</strong>
-          </p>
+          <h2>Resources</h2>
           <ul>
             <li>
               <a
                 href="/assets/resources/test_results_example_10-3-2022.csv"
                 className="usa-link"
               >
-                Standard CSV template with example data
+                SimpleReport spreadsheet template with example data
+              </a>
+            </li>
+            <li>
+              <a href="https://youtu.be/qqhC7PFBdek" className="usa-link">
+                Spreadsheet results uploader training video
```

Comment:
non-blocking: is this a little redundant since it's embedded in the page?

Reply:
Thanks, I'll merge this as it and I would defer this question to @lauraykerr